src/lwc/geTemplateBuilderFormFields/geTemplateBuilderFormFields.js

```diff
@@ -58,15 +71,27 @@ export default class geTemplateBuilderFormFields extends LightningElement {
         let missingRequiredFieldMappings = [];
 
         const elements = this.template.querySelectorAll('lightning-input');
+        let counter = 0;
```

Comment:
Maybe name this variable something more descriptive (countOfConditionallyRequiredFormFields perhaps)?